evap/evaluation/migrations/0098_remove_contributors_from_single_results.py

```diff
@@ -0,0 +1,49 @@
+from django.db import migrations
+
+
+def make_single_result_questionnaire_general(apps, _schema_editor):
+    Questionnaire = apps.get_model('evaluation', 'Questionnaire')
+    Evaluation = apps.get_model('evaluation', 'Evaluation')
+    Contribution = apps.get_model('evaluation', 'Contribution')
+
+    questionnaire = Questionnaire.objects.get(name_en='Single result')
+    questionnaire.is_for_contributors = False
+    questionnaire.save()
+
+    for evaluation in Evaluation.objects.filter(is_single_result=True):
+        contribution = Contribution.objects.get(evaluation=evaluation)
+        contribution.contributor = None
```

Comment:
So we're not using information here since the responsible was also already stored in course.responsibles, correct?

Reply:
not also, but only. after migration 0096 contributions don't hold responsibility information anymore.
and yes, the person from this responsible contribution was then saved as a course responsible.